arrow-core/src/main/kotlin/arrow/core/extensions/eval/apply/EvalApply.kt

```diff
@@ -800,15 +798,15 @@ fun <A, B, C, D, E, FF, G, H, I, Z> Kind<ForEval, Tuple9<A, B, C, D, E, FF, G, H
 @Deprecated(
   "@extension kinded projected functions are deprecated",
   ReplaceWith(
-  "tupled(arg0, arg1)",
-  "arrow.core.Eval.tupled"
+    "Eval.mapN(arg0, arg1) { a, b -> Tuple2(a, b) }",
+    "arrow.core.Eval"
```

Comment:
```suggestion
    "arrow.core.Eval", "arrow.core.Tuple2"
```

Same for other `tupled`/`tupledN` methods